falcosidekick/values.yaml

```diff
@@ -156,8 +156,12 @@ config:
     minimumpriority: ""
 
   gcp:
+<<<<<<< HEAD
```

Comment:
you're rebase or merge failed ^^

Reply:
Ya I just fixed that. I had been using a Mac and switched over to my Windows box and didn't use the right hotkey to save the file :)